SAML2/Core/Signature.hs

```diff
@@ -64,9 +63,5 @@ verifySAMLProtocol b = do
 verifySAMLProtocol' :: SAMLP.SAMLProtocol a => DS.PublicKeys -> XmlTree -> IO a
 verifySAMLProtocol' pubkeys x = do
```

Comment:
i wonder if we can implement the above in terms of this, by passing mempty and parsing the bytestring?

```suggestion
verifySAMLProtocolWithKeys pubkeys x = do
```

?